src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -462,13 +462,30 @@ function dhcpd_radvd_configure($verbose = false, $blacklist = array())
 
         $dnslist = array();
 
+        /* XXX We'll skip this interface unless a valid address can be found for it. */
+
+        $hasipv6 = false; 
         list ($ifcfgipv6, $networkv6) = interfaces_primary_address6($if, $realif, $ifconfig_details);
 
+        if (is_ipaddrv6($ifcfgipv6)) {
```

Comment:
we can skip if going forward makes no sense. I don't like changing the indent / conditional block stuffing of working code. it introduces edge cases...